src/DataTable/tests/TableActions.test.jsx

```diff
@@ -11,155 +12,110 @@ import DataTableContext from '../DataTableContext';
+      const tableInstance = { ...instance, tableActions: [<FirstAction onClick={onClickSpy} />] };
       const wrapper = mount(
         <TableActionsWrapper value={tableInstance} />,
       );
       const button = wrapper.find('button');
       expect(button.length).toEqual(1);
       button.simulate('click');
       expect(onClickSpy).toHaveBeenCalledTimes(1);
-      expect(onClickSpy).toHaveBeenCalledWith({
-        selectedRows: [],
-        tableInstance: expect.any(Object),
-      });
     });
   });
   describe('with two actions', () => {
     const wrapper = mount(<TableActionsWrapper value={{ ...instance, tableActions: twoActions }} />);
     it('displays the user\'s first button as an brand button', () => {
       const buttons = wrapper.find(Button);
       expect(buttons.length).toEqual(2);
-      expect(buttons.get(1).props.variant).toEqual('brand');
+      expect(buttons.get(0).props.variant).toEqual('outline-primary');
     });
     it('displays the user\'s second button as an outline button', () => {
       const buttons = wrapper.find(Button);
-      expect(buttons.get(0).props.variant).toEqual('outline-primary');
+      expect(buttons.get(1).props.variant).toEqual('brand');
     });
     it('reverses the button order so that the primary button is on the right', () => {
       const buttons = wrapper.find(Button);
-      expect(buttons.at(0).text()).toEqual(twoActions[1].buttonText);
-      expect(buttons.at(1).text()).toEqual(twoActions[0].buttonText);
-    });
-    it('passes button classnames', () => {
-      const buttons = wrapper.find(Button);
-      expect(buttons.at(0).props().className).toContain(twoActions[1].className);
-      expect(buttons.at(1).props().className).toContain(twoActions[0].className);
+      expect(buttons.get(1).props.variant).toEqual('brand');
+      expect(buttons.get(0).props.variant).toEqual('outline-primary');
     });
   });
   describe('two actions on click', () => {
     it('performs the primary button action on click', () => {
       const onClickSpy = jest.fn();
-      const tableInstance = { ...instance, tableActions: [{ ...firstAction, handleClick: onClickSpy }, secondAction] };
+      const tableInstance = { ...instance, tableActions: [<FirstAction onClick={onClickSpy} />, <SecondAction />] };
       const wrapper = mount(<TableActionsWrapper value={tableInstance} />);
       const button = wrapper.find(Button).at(1);
       button.simulate('click');
       expect(onClickSpy).toHaveBeenCalledTimes(1);
-      expect(onClickSpy).toHaveBeenCalledWith({
-        selectedRows: [],
-        tableInstance: expect.any(Object),
-      });
     });
     it('performs the second button action on click', () => {
       const onClickSpy = jest.fn();
-      const tableInstance = { ...instance, tableActions: [firstAction, { ...secondAction, handleClick: onClickSpy }] };
+      const tableInstance = { ...instance, tableActions: [<FirstAction />, <FirstAction onClick={onClickSpy} />] };
```

Comment:
nit: Should this be `[<FirstAction />, <SecondAction onClick={onClickSpy} />]`?

Reply:
👍